status-im/status-react
# Add ability to cancel a request to join a community

fixes: https://github.com/status-im/status-mobile/issues/14837

TO TEST:

Go to a community that needs admin approval.

request to join that community.

on the community overview page the three dots in the top right hand corner should bring up a menu with a list of options.

The option in red should be there to "Cancel request to join".

clicking this button should cancel the request for the user and the overview page should reset to how it was beforehand.


Additionally, I updated the mechanism to see what pending requests are available. So to check this is still correct, you can go to the Communities Home page and the 'Pending' tab should have the correct number of pending community requests.
